index1.html

```diff
@@ -18,8 +18,8 @@ <h1>UMB Petitions</h1>
 			<nav>
 				<ul>
 					<li class="current"><a href="index1.html">Create</a></li>
-					<li><a href="sign.html">Sign A Petition</a></li>
-					<li><a href="about.html">About</a></li>
+					<li><a href="Sign.html">Sign A Petition</a></li>
+					<li><a href="About.html">About</a></li>
```

Comment:
Capitalised letters, files started with a capital letter

Reply:
I believe I resolved the issue! Thank you for shedding light on the problem